spec/cache/cache_entry_spec.rb

```diff
@@ -103,33 +103,30 @@
     before { Timecop.freeze now }
 
     it 'returns a hash of the attributes' do
-      expect(cache_entry.to_h).to eq({
-        'value' => cache_entry.value,
-        'created_at' => cache_entry.created_at,
-        'last_accessed_at' => cache_entry.last_accessed_at
-      })
+      expect(cache_entry.to_h).to eq('value' => cache_entry.value,
+                                     'created_at' => cache_entry.created_at,
+                                     'last_accessed_at' => cache_entry.last_accessed_at)
     end
```

Comment:
same thing for indentation.

Please find a Rubocop rule that will enforce the following style:
```ruby
        directory.accounts.create(
          username: 'jlpicard',
          email: "capt#{default_domain}",
          givenName: 'Jean-Luc',
          surname: 'Picard',
          password: 'hakunaMatata179Enterprise'
        )
```